src/js/components/Button/Button.js

```diff
@@ -26,134 +27,129 @@ const isDarkBackground = props => {
-    if (first && second) {
-      contents = (
-        <Box direction="row" align="center" justify="center" gap={gap}>
-          {first}
-          {second}
-        </Box>
-      );
-    } else if (typeof children === 'function') {
-      contents = children({ hover, focus });
-    } else {
-      contents = first || second || children;
-    }
-    // the key events are covered by withFocus()
-    /* eslint-disable jsx-a11y/mouse-events-have-key-events */
-    return (
-      <StyledButton
-        {...rest}
-        as={domTag}
-        ref={forwardRef}
-        aria-label={a11yTitle}
-        colorValue={color}
-        disabled={disabled}
-        hasIcon={!!icon}
-        gap={gap}
-        hasLabel={!!label}
-        fillContainer={fill}
-        focus={focus}
-        href={href}
-        onClick={onClick}
-        onMouseOver={this.onMouseOver}
-        onMouseOut={this.onMouseOut}
-        pad={!plain}
-        plain={
-          typeof plain !== 'undefined'
-            ? plain
-            : Children.count(children) > 0 || (icon && !label)
-        }
-        primary={primary}
-        type={!href ? type : undefined}
-      >
-        {contents}
-      </StyledButton>
+  const domTag = !as && href ? 'a' : as;
+  const first = reverse ? label : buttonIcon;
+  const second = reverse ? buttonIcon : label;
+
+  let contents;
+  if (first && second) {
+    contents = (
+      <Box direction="row" align="center" justify="center" gap={gap}>
+        {first}
+        {second}
+      </Box>
     );
+  } else if (typeof children === 'function') {
+    contents = children({ hover, focus });
+  } else {
+    contents = first || second || children;
   }
-}
+  // the key events are covered by withFocus()
+  /* eslint-disable jsx-a11y/mouse-events-have-key-events */
+  return (
+    <StyledButton
+      {...rest}
+      as={domTag}
+      ref={forwardRef}
+      aria-label={a11yTitle}
+      colorValue={color}
+      disabled={disabled}
+      hasIcon={!!icon}
+      gap={gap}
+      hasLabel={!!label}
+      fillContainer={fill}
+      focus={focus}
+      href={href}
+      onClick={onClick}
+      onMouseOver={onMouseOverButton}
+      onMouseOut={onMouseOutButton}
+      pad={!plain}
+      plain={
+        typeof plain !== 'undefined'
+          ? plain
+          : Children.count(children) > 0 || (icon && !label)
+      }
+      primary={primary}
+      type={!href ? type : undefined}
+    >
+      {contents}
+    </StyledButton>
+  );
+};
+
+Button.propTypes = {
+  type: PropTypes.string,
+  focusIndicator: PropTypes.boolean,
+  gap: PropTypes.string,
+};
+
+Button.defaultProps = {
+  type: 'button',
+  focusIndicator: true,
+  gap: 'small',
+};
 
 Object.setPrototypeOf(Button.defaultProps, defaultProps);
 
 let ButtonDoc;
 if (process.env.NODE_ENV !== 'production') {
-  // eslint-disable-next-line global-require
-  ButtonDoc = require('./doc').doc(Button);
+ ButtonDoc = require('./doc').doc(Button); // eslint-disable-line global-require
```

Comment:
why did this change?

Reply:
This must of been something on my end! Fixed